source/overview/corba.rst

```diff
@@ -0,0 +1,22 @@
+10 things you should know about CORBA
+=====================================
+
+1.  **You don’t need to know CORBA to work with TANGO**
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest **B**\ roker **A**\ rchitecture and it is a standard defined by the `Object Management Group (OMG) <www.omg.org>`_
```

Comment:
@Matteo04052017 , To make an external link works you should always add http:// like:
`Object Management Group (OMG) <http://www.omg.org>`_